### PR TITLE
JSTL-AdminEmailLog

### PR DESCRIPTION
**Pull request by taniach**
@JosephineKwa please review

### CODE REVIEW

**Comment by JosephineKwa** [comment]
It would be best if you could get the `*PageData` variables to have `private` access rather than `public`

**Comment by JosephineKwa** [comment]
Create an init method in the `*PageData` rather than populating it by using `data.*`

**Comment by taniach** [comment]
How about setting the values in the constructor of `pageData`?

**Comment by taniach** [comment]
How about initialising the values in the constructor of `pageData`?

**Comment by JosephineKwa** [comment]
That should be fine too

**Comment by taniach** [comment]
How about initialising in the constructor?

**Comment by taniach** [comment]
@JosephineKwa ready for review

File: src/main/webapp/WEB-INF/tags/statusMessageWithoutFocusingToStatus.tag [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<%@ tag description="Status message" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<c:set var="STATUS_MESSAGE"><%= Const.ParamsNames.STATUS_MESSAGE %></c:set>
+<c:set var="ERROR"><%= Const.ParamsNames.ERROR %></c:set>
+<c:choose>
+    <c:when test="${not empty requestScope[STATUS_MESSAGE]}">
+        <div id="statusMessage"  class="alert alert-<c:choose><c:when test="${requestScope[ERROR]}">danger</c:when><c:otherwise>warning</c:otherwise></c:choose>">
```
Comment (JosephineKwa) [inline src/main/webapp/WEB-INF/tags/statusMessageWithoutFocusingToStatus.tag]: It might be a better idea to use a ternary operator here

Comment (taniach) [inline src/main/webapp/WEB-INF/tags/statusMessageWithoutFocusingToStatus.tag]: Ok. I cherry-picked it from AdminActivityLog but I will change here first.

**Comment by JosephineKwa** [comment]
After addressing the above comments, it should be good to merge :)

**Comment by taniach** [comment]
@JosephineKwa made the changes

**Pull request merged**